File: Example/OpenSwiftUIUITests/UITests/Initialize.swift
//
//  Initialize.swift
//  OpenSwiftUIUITests
//
//  Created by Kyle on 12/29/25.
//

#if OPENSWIFTUI
@_spi(ForTestOnly)
import OpenSwiftUI
#endif

@_cdecl("OpenSwiftUIUITests_InitializeSwift")
func __initialize() -> () {
    #if OPENSWIFTUI
    Color.Resolved._alignWithSwiftUIImplementation = true
    #endif
}

File: Example/OpenSwiftUIUITests/UITests/Initialize.c
//
//  Initialize.c
//  OpenSwiftUIUITests
//
//  Created by Kyle on 12/29/25.
//

#include "Initialize.h"
#include <stdio.h>

extern void OpenSwiftUIUITests_InitializeSwift(void);

static void OpenSwiftUIUITests_Initialize(void) {
    OpenSwiftUIUITests_InitializeSwift();
}
